package.json

```diff
@@ -26,7 +26,9 @@
     "server:start:prod": "npm run start:prod --prefix server",
     "server:test": "npm test --prefix server",
     "start": "concurrently -n server,client \"npm run server:start\" \"npm run client:start\"",
-    "test": "npm run server:test && npm run client:test"
+    "test": "npm run server:test && npm run client:test",
+    "docker-build-base": "docker build -t ghcr.io/plankanban/planka:build-base -f Dockerfile.base .",
+    "docker-build": "docker build -t ghcr.io/plankanban/planka:test -f Dockerfile ."
```

Comment:
I don't understand why these tag names or is it ok?

Reply:
In this case, tag names will stores only local and do not intersect with remote ones. I put it as originals just not to create a mess on dev PCs when using `docker image ls` or so.

Reply:
I’ve changed the names of npm commands a little bit, if you don't mind, so that they were all in the same style. I also added the `-local` suffix to the tag names.